### PR TITLE
feat(import): Add ability to import treesitter groups

### PR DESCRIPTION
**Pull request by ruler501**
Subject to the same experimental status as #109 but should make it easier to start using the new features that introduced.

### CODE REVIEW

File: lua/lush/importer.lua [hunk 1]
```diff
@@ -1,17 +1,26 @@
+local group_pattern = "[@.%w_]+"
```
Comment (rktjmp) [inline lua/lush/importer.lua:1]: `.` should be `%.` for the literal dot?

File: lua/lush/importer.lua [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+
+local function format_group_name(group_name)
+  if (string.match(group_name, "[@.]")) then
```
Comment (rktjmp) [inline lua/lush/importer.lua:4]: again `%.`

File: lua/lush/importer.lua (hunk 1 above)
Comment (ruler501) [reply lua/lush/importer.lua:1]: My understanding is that you don't need to escape inside character groups. From empirical testing it doesn't seem to match arbitrary strings which would happen if it needed to be escaped.

Comment (rktjmp) [reply lua/lush/importer.lua:1]: Yep seems correct!

**Comment by rktjmp** [comment]
:100:

**Pull request merged**